src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/casenotes/CaseNotesReconciliationService.kt

```diff
@@ -226,23 +226,24 @@ data class MismatchCaseNote(
   val offenderNo: String,
   val diffsForDps: Set<String> = emptySet(),
   val diffsForNomis: Set<Long> = emptySet(),
-  var notes: List<String> = emptyList(),
+  val notes: List<String> = emptyList(),
 )
 
-data class CommonCaseNoteFields(
+data class ComparisonCaseNote(
+  val id: String,
   val text: String,
   val type: String,
   val subType: String,
   val occurrenceDateTime: String?,
   val creationDateTime: String?,
-  val dpsUsername: String?,
-  val nomisUsernames: List<String>?,
+  val dpsUsername: String? = null,
```

Comment:
default to `null` for usernames so that we don't need to pass in `null` where it doesn't make sense